endpoints/citrus-http/src/main/java/org/citrusframework/http/message/HttpMessage.java

```diff
@@ -77,6 +77,17 @@ public HttpMessage(final Message message) {
         copyCookies(message);
```

Comment:
this constructor could just contain `this(message, false)`, right? just to have said it.

Reply:
That is - indeed true. I adjusted it accordingly.